native-maven-plugin/src/main/java/org/graalvm/buildtools/maven/AbstractNativeImageMojo.java

```diff
@@ -437,6 +440,21 @@ protected void checkRequiredVersionIfNeeded() throws MojoExecutionException {
         if (requiredVersion == null) {
             return;
         }
+        NativeImageUtils.checkVersion(requiredVersion, getVersionInformation());
+    }
+
+    protected boolean isOracleGraalVM() throws MojoExecutionException {
+        return getVersionInformation().contains(ORACLE_GRAALVM_IDENTIFIER);
```

Comment:
FYI: This is related to a very long time discussion we have on identifying GraalVM properly, and more specifically editions. This has changed several times in the past, this is likely not accurate.

Reply:
@fniephaus Maybe you can chime in on this. We need a (hopefully reliable) way to know if the user is using the EE version of native image. Currently, the `isOracleGraalVM` function does this by calling `native-image --version` (that's the `getVersionInformation()` function) and checking the presence of the string `"Oracle GraalVM"`.